### ISSUE TITLE
Better errors if the JSON file is invalid

### ISSUE THREAD

**Issue by flyon** [toddbluhm/env-cmd#366]
When `.env-cmdrc.json` for example contains an invalid trailing comma then env-cmd gives the following error:

```
 Failed to find .rc file at default paths: [./.env-cmdrc,./.env-cmdrc.js,./.env-cmdrc.json]
```

After lots of trial and error I realised it's because of a trailing comma, but `env-cmd` can perhaps give a more detailed error when it _failed to parse_ a .rc file, instead of saying it cannot find any.

**Comment by k-yle** [comment]
This has actually been fixed already, in #338. I'll link the PR to this ticket...

**Comment by toddbluhm** [comment]
> This has actually been fixed already, in #338. I'll link the PR to this ticket...

Nice catch... I'm still getting back into the swing of things here.
